protobuf/grpc_service.proto

```diff
@@ -1104,6 +1159,17 @@ message ModelStatistics
   //@@     compute).
   //@@
   repeated InferBatchStatistics batch_stats = 7;
+
+  //@@  .. cpp:var:: InferResponseStatistics response_stats (repeated)
+  //@@
+  //@@     The aggregate statistics for different responses that the 
+  //@@     model may produce. The response statistics can provide the
+  //@@     breakdown of the time it takes for the backend to generate
```

Comment:
I don't think you can obtain breakdown for each individual response. These stats are accumulated. At most you can obtain the average of the breakdown across the i'th responses for all the requests that generated same number of responses.

Reply:
Updated.